docs/src/user-guide/core-concepts.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Core Concepts
+layout: doc
+eleventyNavigation:
+    key: core concepts
+    title: Core Concepts
+    parent: user guide
+    order: 2
+---
+
+This page contains a high-level overview of some of the core concepts of ESLint.
+
+## What is ESLint?
+
+ESLint is a configurable JavaScript linter. It's like spell check and autocorrect
```

Comment:
I'd like to put less emphasis on consistent styling and focus on ESLint as a general purpose tool to identify certain patterns in your code. 

Maybe something like "It helps you find and fix problems in your JavaScript code. Problems can be anything from potential runtime bugs, to not following best practices,  to styling issues."?

Reply:
> It helps you find and fix problems in your JavaScript code. Problems can be anything from potential runtime bugs, 
> to not following best practices, to styling issues.

love this. using